lib/hacluster.pm

```diff
@@ -1137,4 +1138,38 @@ sub check_iscsi_failure {
     }
 }
 
+=head3 cluster_status_matches_regex
+
+Remotely check 'crm status' output against regular expression
```

Comment:
Does it run anything remotely or is just about parsing a `crm status` output string no matter where it come from? If it is so maybe the function description can be changed.

Reply:
`output against regular expression` is more about **how** the function does it. Probably the reader is more interested about **what** the function does.

Reply:
refactored